substrate/frame/src/lib.rs

```diff
@@ -230,14 +238,19 @@ pub mod prelude {
 
 	/// Runtime traits
 	#[doc(no_inline)]
-	pub use sp_runtime::traits::{
-		BlockNumberProvider, Bounded, Convert, DispatchInfoOf, Dispatchable, ReduceBy,
-		ReplaceWithDefault, SaturatedConversion, Saturating, StaticLookup, TrailingZeroInput,
+	pub use sp_runtime::{
+		traits::{
+			AtLeast32BitUnsigned, BlockNumberProvider, Bounded, Convert, DispatchInfoOf,
+			Dispatchable, One, ReduceBy, ReplaceWithDefault, SaturatedConversion, Saturating,
+			StaticLookup, TrailingZeroInput, Zero,
+		},
+		BuildStorage, DispatchError, DispatchResult, FixedI64, PerThing, Perbill,
```

Comment:
`DispatchResult` is already imported from frame-support and `DispatchError` is imported below.
```suggestion
		BuildStorage, FixedI64, PerThing, Perbill,
```